### PR TITLE
No self-report

### PR DESCRIPTION
**Pull request by Miky88**
You shoudn't be able to report yourself

### CODE REVIEW

File: commands/report.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ exports.run = async (client, message, args) => {
     return message.channel.send(`${redtick} Please specify an user and a reason`)
 
   const user = client.utils.parseMention(args.shift())
+  if(message.author.id === user) return message.reply("You shoudn't report yourself <:kappalul:527037429161525248>")
```
Comment (NightScript370) [inline commands/report.js]: I don't think we need the Kappa emote

Comment (MoonlightCapital) [inline commands/report.js]: Don't use double quotes, and please put the redtick emote on the response

**Review by MoonlightCapital — CHANGES_REQUESTED** [review]

File: commands/report.js (hunk 1 above)
Comment (Miky88) [reply commands/report.js]: ...done

Comment (Miky88) [reply commands/report.js]: was cute but ok

File: commands/report.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ exports.run = async (client, message, args) => {
     return message.channel.send(`${redtick} Please specify an user and a reason`)
 
   const user = client.utils.parseMention(args.shift())
-  if(message.author.id === user) return message.reply("You shoudn't report yourself <:kappalul:527037429161525248>")
+  if(message.author.id === user) return message.reply(`${redtick}You shoudn't report yourself`")
```
Comment (MoonlightCapital) [inline commands/report.js]: Add a space between the redtick and the text

Remove the double quote at the end

It's shou**l**dn't, not *shoudn't*

**Review by MoonlightCapital — CHANGES_REQUESTED** [review]

File: commands/report.js (hunk 2 above)
Comment (Miky88) [reply commands/report.js]: how i can do thisthose errors

**Review by MoonlightCapital — APPROVED** [review]

**Comment by MoonlightCapital** [comment]
Can you please change message.reply with message.channel.send? I don't like the user being mentioned in responses unless strictly necessary.

**Comment by Miky88** [comment]
ok..

**Comment by Miky88** [comment]
no i deleted patch.. -.- do it yourself :P

**Comment by MoonlightCapital** [comment]
mik cattivo

**Comment by Miky88** [comment]
...

Il lun 15 lug 2019, 11:18 MoonlightCapital <notifications@github.com> ha
scritto:

**Pull request merged**